source/Playnite/MenuItems.cs

```diff
@@ -236,6 +238,16 @@ public static List<SearchItem> GetSearchGameMenuItems(Game game, MainViewModelBa
                 () => model.App.GamesEditor.ToggleHideGame(game))
                 { Icon = game.Hidden ? unHideIcon : hideIcon });
 
+            // Toggle HDR
+            if (HdrUtilities.IsHdrSupported())

```

Comment:
I don't think this is needed. As I mentioned in my previous comment, there is a case where a user might want to change this on a PC that doesn't support HDR.

Reply:
Took this a step further and outright removed this menu item since it was a mistake that I still had it. Therefore, desktop mode can only toggle the HDR feature through the advanced game options dialog.

Just to make sure it is clear now, the only menu item that remains is the menu item for a game in fullscreen mode. And for that item, it only appears if the primary display supports HDR. Do you want me to make it so that the item always appears?